controllers/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -19,18 +22,40 @@ class UserControllerTest
     with Matchers
     with ScalatestRouteTest
     with MockitoSugar
-    with PlayJsonSupport {
+    with PlayJsonSupport
+    with BeforeAndAfterAll {
 
   private val userService = mock[UserService]
   private val userController = new UserController(userService)
+  private var dummyUser: User = _
```

Comment:
I have already seen dummy user with beforeAll somewhere else(I've commented it) can we combine it?(DRY)

Reply:
Done